### DISCUSSION TITLE
Accelerator list

### DISCUSSION THREAD

**Discussion by Oqwert** [Open-Shell/Open-Shell-Menu#1964] [Q&A] [answered]
I recently accidentally shut down my computer twice by attempting to search "audio" without having the search bar selected because my last click was on "All Programs". It seems "a" selects the shut down menu and "u" causes the computer to shut down. Is there a list of all default accelerators anywhere? Apologies if the answer is obvious, but I did try to look around here, in the help menu and in the settings menu and could not find anything other than an option to disable them.

**Comment by ge0rdi** [comment, marked as answer]
> Is there a list of all default accelerators anywhere?

You can press `Alt` key to show accelerators (underscore).
For menu lists (such as `Apps`) accelerators are usually assigned by the items first letter (or subsequent letters if some are taken already).

> could not find anything other than an option to disable them

You have couple of options here:

- search for `accelerators` in settings - there you can disable them completely, or require `Alt` key to be pressed in order to trigger accelerator (Windows default behavior)
- look for `show search box` setting - you can select `Access normally` there so that search box will be focused when you open Start menu and you can search right away

**Reply by Oqwert** [reply]
Thanks, Alt did the trick. I do have "Access normally" already enabled but the search box still gets deselected if the search is done after clicking on "All Programs" for an initial check and sometimes I forget to reselect it.
